package.json

```diff
@@ -32,15 +35,14 @@
     "lint:ts:check": "npx nx run-many --target=lint:ts:check",
     "lint:check": "npx nx run-many --target=lint:check",
     "lint:fix": "npx nx run-many --target=lint:fix",
-    "lint:specs:fix": "yarn run markdownlint-cli2-fix \"./specs/**/*.md\"",
-    "lint:specs:check": "yarn run markdownlint-cli2  \"./specs/**/*.md\"",
-    "lint:specs:toc": "yarn run doctoc '--title=**Table of Contents**' ./specs",
+    "lint:specs:fix": "npx markdownlint-cli2-fix \"./specs/**/*.md\"",
+    "lint:specs:check": "npx markdownlint-cli2  \"./specs/**/*.md\"",
+    "lint:specs:toc": "npx doctoc '--title=**Table of Contents**' ./specs",
```

Comment:
Since we're using `pnpm`, we should migrate away from `npx` and use `pnpm dlx` instead.
See: https://pnpm.io/cli/dlx

Reply:
Oh this is dope because it would save space on our machine compared to npx.  I'll actually do this in it's own pr.  TIL